### DISCUSSION TITLE
I can't connect my laravel project on windows with PlanetScale

### DISCUSSION THREAD

**Discussion by Tommy-S05** [planetscale/discussion#528] [FAQs] [answered]
when i run the migrations i get the next error:  PDOException::("failed loading cafile stream: `/etc/ssl/certs/ca-certificates.crt'")

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @Tommy-S05,

We have some more info here: https://planetscale.com/docs/concepts/secure-connections#windows

The TLDR though is:
1. Go here: https://curl.se/docs/caextract.html
2. Download that top link: [`cacert.pem`](https://curl.se/ca/cacert.pem)
3. In your project, update the cert path in your `.env` to the file you just downloaded. `MYSQL_ATTR_SSL_CA=cert/path/here/cacert.pem`

That should get you connected!
